docs/getting-started/raspberry-pi-raspberryos-quick-start.md

```diff
@@ -74,29 +74,35 @@ following steps:
     sudo apt-get install ./kura_<version>_raspberry-pi_installer.deb
     ```
 
-6. It could happen that `wlan` interface is "soft blocked" by default and needs to be enabled. To see if it is blocked run:
+6. For a correct configuration of the Wlan interface, it is necessary to set the **Locale** and the **WLAN Country** through the `raspi-config` command:
 
     ```
-    rfkill list
+    sudo raspi-config
     ```
 
-    and unblock it with:
+    ![raspi-config menu](./images/raspi-config.png)
 
-    ```
-    sudo rfkill unblock wlan
-    ```
+    From the raspi-config main menu select `Localisation Options`:
+
+
+    ![raspi-config menu](./images/raspi-localisation-menu.png)
+
+    Then modify the following settings with the value in the table:
```

Comment:
Maybe not the exact values in this table if they're not in Italy. This table could be an *example* of the values to set.